### DISCUSSION TITLE
1, 2, Refactor! เมื่อไหร่ที่ควร Refactor? | First Sutham

### DISCUSSION THREAD

**Discussion by giscus** [heyfirst/heyfirst.co#8] [General]
# 1, 2, Refactor! เมื่อไหร่ที่ควร Refactor? | heyfirst.co

กฏข้อนี้ถูกพูดถึงในหัวข้อ When Should We Refactor? ในหนังสือครับ ผมเลยหยิบเอาส่วนหนึ่งของในหนังสือมาให้อ่านกัน

https://heyfirst.co/blog/1-2-refactor

**Comment by heyfirst** [comment]
อู้ว
